source/declarations.tex

```diff
@@ -2921,7 +2921,7 @@
 Because a \grammarterm{using-declaration} designates a base class member
 (and not a member subobject or a member function of a base class
 subobject), a \grammarterm{using-declaration} cannot be used to resolve
-inherited member ambiguities. For example,
+inherited member ambiguities by itself. For example,
```

Comment:
I suppose what I was trying to get at was the following:
struct A { int x(); };
struct B : A { int x(B); };
struct C : A {  using A::x; int x(C); };
struct D: B, C {
  using C::x;
  using B::x;
};
D d;
d.x(C{}); // OK - because using declaration helps resolve the inherited member lookup ambiguities
d.x(); // NOT OK - because of conversion of D to base class
d.C::x(); // OK - using declaration brings x() into C's scope and and A is unambiguous base of C which is unambiguous of D

Does this seem reasonable - or should we drop the 'by itself'